rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-utility.yml

```diff
@@ -70,3 +70,21 @@
     msg: "One or more of the cinder services are in the down state. Check the output above for more detail."
   when: "'down' in '{{ item.split(' ')[-1] }}'"
   with_items: "{{ cinder_output.stdout_lines | default([]) }}"
+
+# Adds orchestration service checks per https://rpc-openstack.atlassian.net/browse/UG-543
+- name: Grab relevant output of orchestration service list
+  shell: |
+    . ~/openrc
+    openstack orchestration service list --insecure
+  register: orchestration_output
+  failed_when: "orchestration_output.rc != 0"
+
+- name: Display output of orchestration service list
+  debug:
+    var: orchestration_output
+
+- name: Fail if any of the orchestration services are down
+  fail:
+    msg: "One or more of the orchestration services are down. Check the output above for more detail."
+  when: "'down' in '{{ item.split(' ')[-1] }}'"
```

Comment:
Since this is the second review recommending performing this test differently, I may need to change my stance. Is there a value in running this test in a different way than the existing nova, cinder, and neutron services are being checked?  For simplicity, I matched the three other tests so they would be logically grouped reading through them, being able to see each one is using the same method for testing.  Parsing JSON does make perfect sense and is the way I would have gone with it if not for the existing service checks using string parsing.